microservices/site-launch/shared/types.ts

```diff
@@ -38,3 +38,44 @@ export interface SiteLaunchMessage {
   status?: SiteLaunchStatus
   statusMetadata?: string
 }
+
+export function isSiteLaunchMessage(obj: unknown): obj is SiteLaunchMessage {
+  if (typeof obj !== "object" || obj === null) {
+    return false
+  }
+
+  const message = obj as SiteLaunchMessage
+
+  return (
+    typeof message.repoName === "string" &&
```

Comment:
actually what i had in mind was just checking `!!message.<required_property>` but this works too HAHA thx for ur hard work